rest/server_context.go

```diff
@@ -815,21 +815,44 @@ func (sc *ServerContext) installPrincipals(context *db.DatabaseContext, spec map
 		} else {
 			princ.Name = &name
 		}
-		_, err := context.UpdatePrincipal(*princ, (what == "user"), isGuest)
-		if err != nil {
-			// A conflict error just means updatePrincipal didn't overwrite an existing user.
-			if status, _ := base.ErrorAsHTTPStatus(err); status != http.StatusConflict {
-				return err
+
+		worker := func() (shouldRetry bool, err error, value interface{}) {
+			_, err = context.UpdatePrincipal(*princ, (what == "user"), isGuest)
+			if err != nil {
+				if base.IsConflictError(err) {
```

Comment:
Can we use the existing IsCasMismatch instead of IsConflictError?  They seem redundant.

Reply:
Fixed.  Also I removed un unused param from `IsCasMismatch()`, which requires a companion change to SG Accel.